src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,125 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int DEFAULT_CAPACITY = 16;
+    private int size;
+    private Node<K, V>[] table = new Node[DEFAULT_CAPACITY];
+    private Node<K, V> nullKeyNode;
 
     @Override
     public void put(K key, V value) {
-
+        if (getThreshold() >= size) {
```

Comment:
The condition `getThreshold() >= size` is incorrect. It should be `size >= getThreshold()` to ensure the resizing happens when the number of elements exceeds the threshold, not the other way around.